### PR TITLE
W 11793961 http oauth updates esb

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc [hunk 1]
```diff
@@ -280,6 +361,36 @@ a|
 
 |===
 
+==== Shopify
+[%header%autowidth.spread]
+
+|===
+
+|Step |Field |Example Value |Notes
+.9+|Connect to your HTTP Account |*Connection Display Name* | `+Shopify at dpelaez+shopifydev@mulesoft.com (QA)+` | For more information, see the https://partners.shopify.com/[Shopify Partner Program^].
```
Comment (ignamiguel) [inline en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc:370]: This should be
`Shopify at my.account@mail.com`

**Review by ignamiguel — CHANGES_REQUESTED** [review]
we should remove all references to personal accounts such as `dpelaez` and `@gmail`

**Review by ignamiguel — COMMENTED** [review]
@emileeburkeVT could you share the link where this information is published?

we should also change all the client_secrets to make it clear that those values are just examples and not real secrets

**Review by ignamiguel — COMMENTED** [review]
I also don't see the new feature: "Customization of the Authorization Header"


We're introducing these new fields:

- Authorization Header Name
- Authorization Header Value Prefix
- Authorization Header Custom Value Prefix

### Authorization Header Name
Default: `Authorization`
Type: string
Header key name that will be sent with the request.

### Authorization Header Value Prefix
Default: Bearer Prefix
Options: [Bearer Prefix, No Prefix, Custom Prefix]
Header value prefix used along with the access token

### Authorization Header Custom Value Prefix
Custom Value Prefix
If we select "Custom Prefix" this string will be used

image

File: en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc (hunk 1 above)
Comment (lipusal) [reply en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc:370]: May I suggest `my.account@example.com`? Makes it even more obvious that it's an example

**Comment by lipusal** [comment]
Agree with @ignamiguel. A few suggestions to his suggestions (suggestions²). Some of these suggestions come from reading [RFC 2616 section 4.2](https://www.rfc-editor.org/rfc/rfc2616#section-4.2) which describes HTTP headers as 
> Each header [...] consists of a name followed by a colon (":") and the field value.

So the term "name" is preferable over "key name" or "key".

@emileeburkeVT it might be better for us to wait for you to introduce the new section, so we can leave comments on the code itself instead of making a comment chain. Anyway, my suggestions:

---

### Authorization Header Name
Default: `Authorization`
Type: string
Name of the authorization header.

### Authorization Header Value Prefix
Default: `Bearer`
Options: [Bearer Prefix, No Prefix, Custom Prefix]
Prefix used in the value of the authorization header.

### Authorization Header Custom Value Prefix
Custom Value Prefix
Custom prefix value.

File: en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc (hunk 1 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc:370]:
```suggestion
.9+|Connect to your HTTP Account |*Connection Display Name* | `+Shopify at my.account@example.com (QA)+` | For more information, see the https://partners.shopify.com/[Shopify Partner Program^].
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc [hunk 2]
```diff
@@ -194,6 +194,45 @@ Parameter 2:
...
+|*Client Secret* |`Yf8q-wbEJjeJIhx5iuE-BiydpL1oCj7wVy2gaoYDEMCe4xwWO_Q2vbjyJg6rGmD7L` |
+|*Scopes* a|
+
+* `offline_access`
+* `read:jira-work`
+* `manage:jira-project`
+* `read:jira-user`
+* `write:jira-work` |
+
+|*Authorization URL* | `+https://auth.atlassian.com/authorize+` | You must register your callback URL when creating a Jira account.  As a result, these these credentials only work in staging environments.
```
Comment (bjoson-mulesoft) [inline en/mulesoft/modules/ROOT/pages/ms_composer_http_reference.adoc:216]: these these

**Review by bjoson-mulesoft — APPROVED** [review]

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**